core/src/main/scala/org/apache/spark/ui/WebUI.scala

```diff
@@ -101,12 +101,12 @@ private[spark] abstract class WebUI(
   }
 
   /**
-   * Add a handler for static content.
+   * Adds a handler for static content.
```

Comment:
In this file, `s` is not added.

Reply:
@jaceklaskowski please address this super tiny issue.